File: swift/model/A/ClassB.swift

import UIKit


class ClassB{

    var var3 = Int()
    var var4 = String()


    init(){}

    init(dict:Dictionary<String, String>){
        self.var3 = Int(dict["var3"]!)!
        self.var4 = dict["var4"]!
    }

    func setVar3(var3:Int){
        self.var3=var3
    }

    func getVar3()->Int{
        return self.var3
    }

    func setVar4(var4:String){
        self.var4=var4
    }

    func getVar4()->String{
        return self.var4
    }

    func toDict()->Dictionary<String, String>{
        var dict = Dictionary<String, String>()
        dict["var3"] = String(self.var3)
        dict["var4"] = String(self.var4)
        return dict
    }
}

File: java/model/A/ClassB.java
package model.A;

import java.io.Serializable;
import org.json.*;
import java.util.*;

public class ClassB implements Serializable{

    public ClassB(){}

    private int var3 = 0;
    private String var4 = "";


    public ClassB(JSONObject dict) throws JSONException{
        this.var3 = dict.getInt("var3");
        this.var4 = dict.getString("var4");
    }

    public void setVar3(int var3){
        this.var3=var3;
    }

    public int getVar3(){
        return this.var3;
    }

    public void setVar4(String var4){
        this.var4=var4;
    }

    public String getVar4(){
        return this.var4;
    }

    public JSONObject toDict() throws JSONException{
        JSONObject dict = new JSONObject();
        dict.put("var3", String.valueOf(this.var3));
        dict.put("var4", String.valueOf(this.var4));
        return dict;
    }
}
